src/Main.java

```diff
@@ -0,0 +1,173 @@
+// 1. Maak een main-klasse aan in de src-map.
+public class Main {
+// 2. Maak vervolgens binnen de main-klasse een main-methode aan. Anders kun je de applicatie immers niet draaien!
+    public static void main(String[] args) {
+// 5. Instantieer in de main methode jouw 4 verschillende Pokèmon met de verschillende attributen.
+        ElectricPokemon pokemon1 = new ElectricPokemon("Pikachu", 10, 100, 0);
+        FirePokemon pokemon2 = new FirePokemon("Charizard", 50, 200, 0, 80);
+        WaterPokemon pokemon3 = new WaterPokemon("Blastoise", 60, 250, 0, 90);
+        GrassPokemon pokemon4 = new GrassPokemon("Venusaur", 55, 220, 0, 85);
+
+// 6. Voer in de main methode verschillende methodes uit van verschillende Pokemons, uit zowel de subklassen als uit de super klasse.
+        pokemon1.thunderPunch();
+        pokemon2.inferno();
+        pokemon3.hydroPump();
+        pokemon4.solarBeam();
+    }
+}
+// 3. Maak een Pokèmon Super klasse aan en zorg ervoor dat deze voldoet aan de gestelde randvoorwaarden
+class Pokemon {
```

Comment:
Dit werkt, maar het is beter en overzichtelijker om nieuwe classes in een eigen bestand te zetten. De Pokemon klasse zou dan in "Pokemon.java" komen. In de afgelopen huiswerkles heeft John ook nog eens laten zien hoe je nieuwe Class bestanden maakt in IntelliJ